### PR TITLE
Pseudocode clarification in README and adding new rule to .gitignore 

### PR DESCRIPTION
**Pull request by ionafan2**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,5 @@
 composer.lock
 vendor/
+
+# Covers JetBrains IDEs: IntelliJ, RubyMine, PhpStorm, AppCode, PyCharm, CLion, Android Studio
+.idea/
```
Comment (stof) [inline .gitignore]: IMO, this does not belong in the project ignore rules. You should ignore it in your dev environment instead (either globally or per project): https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

Comment (ionafan2) [inline .gitignore]: sorry, it was for develop branch in my repo
and thanks for helpful advice )
fixed

**Comment by tgalopin** [comment]
Do you think it's unclear that `$sender` is a PersonInterface? I think it's quite easy to understand, and as it uses the same naming convention than `$recipient` I'm not sure it's a good idea to change it. What do you think?

**Comment by ionafan2** [comment]
@tgalopin this lines is little bit confusing me when I was reading them first time

``` php
$sender = new \FOS\Message\Sender($repository, $driver);
$sender->startConversation($sender, $recipient, $body, $subject = null);
```

as I understood this `$sender = new \FOS\Message\Sender($repository, $driver);` is not a PersonInterface. so I thought it will be more clear if we change name of variable `$sender` in `startConversation` params

was i wrong?
anyway it is just for Pseudocode in README file

**Comment by tgalopin** [comment]
Oh, you're completely right :) . Can you update `$recipient` to `$recipientPerson` to have the same naming convention? Then I'll merge that.

**Comment by tgalopin** [comment]
Thanks @ionafan2 !

**Comment by ionafan2** [comment]
Not at all )

**Pull request merged**